src/UI/Internal/Tables/View.elm

```diff
@@ -32,7 +31,15 @@ cellContentRender renderConfig cell_ =
 simpleText : RenderConfig -> Text -> Element msg
 simpleText renderConfig text =
     text
-        |> EllipsizableTooltip.view renderConfig
+        |> Text.setEllipsis True
```

Comment:
Should we use Keyed.el here in order to avoid unnecessary redraws? 🤔

Reply:
wouldn't work, the cols/rows until here should have keyed while this one should use Lazy.el